src/main/java/com/minecolonies/coremod/generation/LootTableBuilder.java

```diff
@@ -0,0 +1,90 @@
+package com.minecolonies.coremod.generation;

```

Comment:
Should be part of the DataGen lib.
Or use the default one provided by Minecraft itself.

Reply:
Could do, although this is mostly a convenience wrapper for use in the MC-only datagen process, so it's a bit borderline.

As previously mentioned I'm not using the MC builder directly since it's write-only and the sifter datagen needs to be able to read back the data.